config/routes.rb

```diff
@@ -6,7 +6,11 @@
   # All the Api endpoints must be under /api/v1 and must have an extension .json.
   namespace :api do
     namespace :v1 do
-      resources :sessions, only: %i[index create destroy]
+      resources :sessions, only: %i[index create] do
+        collection do
+          delete 'sign_out', to: 'sessions#destroy'
```

Comment:
Is `resources :sessions, only: %i[index create destroy]` isn't suiting your needs?

Reply:
Seems standard for destroying a session.
Default URI for destroy is /api/v1/sessions/:id
I figured we would rather not be passing the session id around.

Reply:
You got reason,